lib/dynamic-files-renderer.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Rendering context.
```

Comment:
RenderingContext is a reusable type. It is used in [/lib/modules/handlebars/handlebars.js#L667](https://github.com/sajithar/carbon-uuf/blob/master/lib/modules/handlebars/handlebars.js#L667), [/lib/units-router.js#L35](https://github.com/sajithar/carbon-uuf/blob/master/lib/units-router.js#L35), [/lib/units-router.js#L35](https://github.com/sajithar/carbon-uuf/blob/master/lib/units-router.js#L35) & etc.